src/StelLogger.cpp

```diff
@@ -214,11 +214,7 @@ void StelLogger::init(const QString& logFilePath)
 	dmesg.waitForStarted();
 	dmesg.waitForFinished();
 	const QString dmesgData(dmesg.readAll());
-	#if (QT_VERSION>=QT_VERSION_CHECK(5, 14, 0))
```

Comment:
Not sure why you removed the branches. I think it may have to be reversed though. Qt::SkipEmptyParts was introduced in Qt5.14. (https://doc.qt.io/qt-5/qt.html#SplitBehaviorFlags-enum). Is Q_OS_BSD4 guaranteed to build on Qt5.14+?

Reply:
You can ignore this part, because in the branch "refactoring/logger" I've rewriting the part of logger for getting hardware info without calling CLI tools.